InTune/InTune/ViewControllers/LoginViewController.swift

```diff
@@ -14,6 +14,7 @@ class LoginViewController: UIViewController {
         super.viewDidLoad()
 
         // Do any additional setup after loading the view.
+        view.backgroundColor = .blue
```

Comment:
Just a thought,
If possible we should set color and fonts in the interface builder. It will help us in maintaining the code and consistency, not a big deal :)